Upstream: Unknown
Reason: fixes compliation with gcc-10 and -fno-common

--- a/src/parseopt.c
+++ b/src/parseopt.c
@@ -255,8 +255,8 @@
 }
 
 char *parseopt_program_name;
-char *parseopt_program_doc;
-char *parseopt_program_args;
+extern char *parseopt_program_doc;
+extern char *parseopt_program_args;
 const char *program_bug_address = "<" PACKAGE_BUGREPORT ">";
 void (*parseopt_help_hook) (FILE *stream);
 

